partiql-lang/src/main/kotlin/org/partiql/lang/eval/builtins/ScalarBuiltinsSql.kt

```diff
@@ -350,6 +359,70 @@ internal object ExprFunctionUpper : ExprFunction {
     }
 }
 
+/**
+ * This function is to test overloading.
+ */
+internal object ExprFunctionPower : ExprFunction {
```

Comment:
While this was good for testing, we don't want to introduce any of the `"query_power"` functions to the Scalar Builtins. We can delete these objects.